flair/datasets/base.py

```diff
@@ -22,55 +20,23 @@ def __init__(
         shuffle=False,
         sampler=None,
         batch_sampler=None,
-        num_workers=None,
         drop_last=False,
         timeout=0,
         worker_init_fn=None,
```

Comment:
Good point. The only remaining difference is that the `collate_fn` is set to `list`, which is necessary since we use `DataLoader` to return batches that are lists of `DataPoint`. I guess the options are either to remove this class and put the custom collate_fn in each call of DataLoader or keep it as it is.

Reply:
I wrongly assumed list would be the default in torch as well.